### PR TITLE
Add externalDocs to root as per spec

### PR DESCRIPTION
**Pull request by gabrieltaylor**
Resolves https://github.com/fotinakis/swagger-blocks/issues/51

### CODE REVIEW

File: spec/lib/swagger_v2_blocks_spec.rb [hunk 1]
```diff
@@ -38,6 +38,9 @@ class PetControllerV2
         key 'read:pets', 'read your pets'
       end
     end
+    externalDocs description: 'Find more info here' do
```
Comment (fotinakis) [inline spec/lib/swagger_v2_blocks_spec.rb]: This should be `external_docs` here and the method name.

**Comment by fotinakis** [comment]
Thanks @gabrieltaylor, one comment here.

**Comment by gabrieltaylor** [comment]
@fotinakis I thought it should be `external_docs` too but the existing function is camelCase so I just did the same for this one to be consistent

https://github.com/fotinakis/swagger-blocks/blob/48a2ad05da6d8f658e1d9f4d88cb360eca071e22/lib/swagger/blocks/nodes/tag_node.rb#L9

Thoughts?

**Comment by fotinakis** [comment]
Hmm yes you're right, that is consistently inconsistent. :) I'll file an issue to update that in a future release, this looks good as is.

**Comment by fotinakis** [comment]
Released in swagger-blocks 2.0.2. Thanks for contributing!

**Pull request merged**